src/test/java/com/example/amazoncom/BrowserStackSignUpPage.java

```diff
@@ -0,0 +1,59 @@
+package com.example.amazoncom;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class BrowserStackSignUpPage {
+
+  WebDriver driver;
+  @FindBy(xpath = "//a[contains(@data-nav-role,'signin')]")
+  WebElement signInLink;
+  @FindBy(xpath = "//*[@id='ap_email']")
+  WebElement businessEmail;
+  @FindBy(xpath = " //input[@id=\"continue\"]")
+  WebElement buttonToLogging1;
+  @FindBy(xpath = " //input[@id=\"signInSubmit\"]")
+  WebElement buttonEntrance;
+  @FindBy(xpath = "//input[@type='password']")
+  WebElement password;
+  @FindBy(xpath = "//span[contains(@id,'nav-link')]")
+  WebElement textIamInside;
+
+  public BrowserStackSignUpPage(WebDriver driver) {
+    this.driver = driver;
+    PageFactory.initElements(driver, this);
+  }
+
+  public void textIamInside() {
```

Comment:
the name of the functions does not reflect it has assertion inside.
and pls try not to do assertions on the page object level.